tests/cargo-kani/firecracker-block-example/README.md

```diff
@@ -0,0 +1,38 @@
+This example accompanies Kani's post on Firecracker. This describes a proof
+harness for ensuring that the Firecracker block device `parse` method adheres
+to a virtio requirement (see below). We implement this as a standalone example
+with some simplifications (search for "Kani change" in the source).
+
+This example is based on code from Firecracker. In particular,
+
+  - https://github.com/firecracker-microvm/firecracker/tree/main/src/devices/src/virtio/block
+  - https://github.com/firecracker-microvm/firecracker/blob/main/src/devices/src/virtio/queue.rs
```

Comment:
```suggestion
  - <https://github.com/firecracker-microvm/firecracker/tree/main/src/devices/src/virtio/block>
  - <https://github.com/firecracker-microvm/firecracker/blob/main/src/devices/src/virtio/queue.rs>
```
Seems to be the standard way to [add links in markdown](https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses)